iaaip/index.html

```diff
@@ -0,0 +1,160 @@
+
+
+
+<a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+
+
+
+
+<div id="cookie-banner" style="display: none;">
+  <div class="govuk-width-container">
+    <p>GOV.UK uses cookies to make the site simpler. <a href="/cookie-policy" class="govuk-link" title="Find out more about cookies">Find out more about cookies</a></p>
+  </div>
+</div>
+
+<header class="govuk-header " role="banner" data-module="govuk-header">
+  <div class="govuk-header__container govuk-width-container">
+    <div class="govuk-header__logo">
+        <span class="govuk-header__logotype">
+          <img src="/assets/images/hmcts-logo-compress.svg" alt="HM Courts and Tribunals Service">
+        </span>
+    </div>
+    <div class="govuk-header__content govuk-header__content--flex">
+      <a href="/start-appeal" class="govuk-header__link govuk-header__link--service-name">
+        Appeal an immigration or asylum decision
+      </a>
+
+      <a href="/logout" class="govuk-header__link govuk-header__link--service-name govuk-header__link--signout">
+        Sign out
+      </a>
+
+    </div>
+  </div>
+</header>
+
+
+
+
+<div class="govuk-width-container ">
+
+  <div class="govuk-phase-banner">
+    <p class="govuk-phase-banner__content">
+      <strong class="govuk-tag govuk-phase-banner__content__tag">
+        Beta
+      </strong>
+      <span class="govuk-phase-banner__text">
+      This is a new service – your <a class="govuk-link" href="#" target="_blank">feedback</a> will help us to improve it. (opens in a new window)
+    </span>
+    </p>
+  </div>
+
+
+
+  <main class="govuk-main-wrapper " id="main-content" role="main">
+
+
+
+    <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
+    <p class="govuk-body">Try again later.</p>
+    <h2 class="govuk-body govuk-body govuk-!-font-weight-bold">Contact us for help</h2>
+    <p class="govuk-body">Telephone: 0300 123 1711</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Lines open Monday to Friday 8:30am to 5pm</p>
+    <p class="govuk-body">
+      <a class="govuk-link" href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>
+    </p>
+
+
+
+
+  </main>
+</div>
+
+
+
+<footer class="govuk-footer " role="contentinfo">
+  <div class="govuk-width-container ">
+
+    <div class="govuk-footer__meta">
+      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+
+        <h2 class="govuk-visually-hidden">Support links</h2>
+
+        <ul class="govuk-footer__inline-list">
+
+          <li class="govuk-footer__inline-list-item">
+            <a class="govuk-footer__link" href="https://hmcts-access.service.gov.uk/contact-us">
+              Contact Us
+            </a>
+          </li>
+
+          <li class="govuk-footer__inline-list-item">
+            <a class="govuk-footer__link" href="/cookie-policy">
+              Cookies
```

Comment:
Don't think we should have any of these links in the footer as they will be dead and redirect back to this page.